zebra-chain/src/parameters/network_upgrade.rs

```diff
@@ -85,7 +85,7 @@ pub(crate) const TESTNET_ACTIVATION_HEIGHTS: &[(block::Height, NetworkUpgrade)]
     (block::Height(584_000), Blossom),
     (block::Height(903_800), Heartwood),
     (block::Height(1_028_500), Canopy),
-    // TODO: Add Nu5 testnet activation height
+    (block::Height(1_590_000), Nu5),
```

Comment:
See the corresponding `zcashd` change at:
https://github.com/zcash/librustzcash/pull/447/files#diff-390655b2b7dbbfda727fa47b40e98c43fd479427f293da891c00b708ba01c9b7R243

```suggestion
    (block::Height(1_599_200), Nu5),
```